src/api/PacbioRun.js

```diff
@@ -12,9 +13,10 @@ const buildWell = (row, column, generate_hifi='', pre_extension_time='2') => ({
     ccs_analysis_output: '',
     libraries: [],
     pre_extension_time
+    // pre_extension_time: preExtenstionTimeDefault,
```

Comment:
There seem to be a few bits commented out. Don't know if they still need to be here?